app/cdap/components/AddTransformation/constants.ts

```diff
@@ -40,3 +40,15 @@ export const multipleColumnSelected = [
     isMoreThanTwo: true,
   },
 ];
+
+export const directiveComponentDefaultValues = {
```

Comment:
please rename the variable name to relate to trasnformations
plz dont use directives keyword for transformation functions or data